lib/ui/window/platform_configuration.cc

```diff
@@ -77,12 +86,43 @@ void PlatformConfiguration::DidCreateIsolate() {
   report_timings_.Set(tonic::DartState::Current(),
                       Dart_GetField(library, tonic::ToDart("_reportTimings")));
 
-  // TODO(loicsharma): This should only be created if the embedder enables the
-  // implicit view.
-  // See: https://github.com/flutter/flutter/issues/120306
-  windows_.emplace(kImplicitViewId,
-                   std::make_unique<Window>(
-                       kImplicitViewId, ViewportMetrics{1.0, 0.0, 0.0, -1, 0}));
+  library_.Set(tonic::DartState::Current(),
+               Dart_LookupLibrary(tonic::ToDart("dart:ui")));
```

Comment:
Can we reuse the `library` created on [line 48](https://github.com/flutter/engine/pull/42991/files#diff-d9b11435542f935fb2c739d1abdf3c8f92b5f3bab797e614ea74324e52c91b98R48)?

```suggestion
               library);
```